Standards/Configuration/.editorconfig

```
@@ -204,13 +220,33 @@ dotnet_naming_rule.interface_types_must_be_prefixed_with_i.severity = error
 dotnet_naming_rule.interface_types_must_be_prefixed_with_i.symbols = interface_types
 dotnet_naming_rule.interface_types_must_be_prefixed_with_i.style = prefix_interface_interface_with_i
 
+csharp_style_namespace_declarations= file_scoped:warning
```

```suggestion
csharp_style_namespace_declarations = file_scoped:warning
```

and a question: could we make it an error? Does someone actually want block scopes?